model/SystemStatus/AbstractSystemStatusService.php

```diff
@@ -114,4 +117,55 @@ protected function getChecks(): array
      * @return string
      */
     abstract protected function getChecksType(): string;
+
+    /**
+     * @return string
+     */
+    public function getInstanceId(): string
+    {
+        if ($this->instanceId !== null) {
+            return $this->instanceId;
+        }
+
+        //here we assume that ths is instance inside AWS stack.
+        if ($this->getServiceLocator()->has('generis/awsClient')) {
+            try {
+                $ch = curl_init();
+                curl_setopt($ch, CURLOPT_URL, 'http://169.254.169.254/latest/meta-data/instance-id');
```

Comment:
Could we create a constant for this IP? It will be very helpful in the future if we will need to change it.